src/java.base/share/classes/java/lang/Class.java

```diff
@@ -415,15 +415,15 @@ private static Class<?> forName(String className, Class<?> caller)
      *
      * <p> For example, in an instance method the expression:
      *
-     * <blockquote>
-     *  {@code Class.forName("Foo")}
-     * </blockquote>
+     * {@snippet lang="java" :
+     *  Class.forName("Foo")
+     * }
      *
      * is equivalent to:
      *
-     * <blockquote>
-     *  {@code Class.forName("Foo", true, this.getClass().getClassLoader())}
-     * </blockquote>
+     * {@snippet lang="java" :
+     *  Class.forName("Foo", true, this.getClass().getClassLoader())
+     * }
```

Comment:
Not important but there are at least 3 different indentation widths in the update. Does it look okay in the generated javadoc?

Reply:
The previous blockquote tag normalized the space -- adjusted the leading whitespace to be consistent.